vue/sbc-common-components/tests/unit/logout.spec.ts

```diff
@@ -48,14 +39,12 @@ describe('SbcHeader.vue', () => {
     expect(cmp.find('.user-name').text().startsWith('BCREGTEST')).toBeTruthy()
   })
 
-  it('logout/in button click invokes logout method', () => {
+  it('logout method is called when logout button is clicked', async () => {
     const stub = vitest.fn()
     cmp.setMethods({ logout: stub })
-    cmp.find('.user-account-btn').trigger('click')
-    expect(cmp.find('.v-list-item__title')).toBeTruthy()
```

Comment:
After upgrade, @vue/test-utils maybe not bound events correctly, it can't find <template #activator="{ on }">